package.json

```diff
@@ -27,7 +27,7 @@
     "ccxt": "^1.10.171",
     "express": "^4.16.2",
     "forex.analytics": "mkmarek/forex.analytics#7bc278987700d4204e959af17de61495941d1a14",
-    "gdax": "^0.4.2",
+    "gdax": "https://github.com/coinbase/gdax-node.git",
```

Comment:
I'd use:
`coinbase/gdax-node#v0.5.0`

And you still need to commit an updated `package-lock.json`